src/configuration/services/mysql.md

```diff
@@ -257,3 +254,25 @@ platform sql --relationship database -e master < my_database_snapshot.sql
 > **note**
 > Importing a database snapshot is a destructive operation. It will overwrite data already in your database.
 > Taking a snapshot or a database export before doing so is strongly recommended.
+
+## Troubleshooting
+
+### definer/invoker of view lack rights to use them
+
+There is a single MySQL user, so you can not use "DEFINER" Access Control mechanism for Stored Programs and Views.
+
+When creating a VIEW, you may need to explicitly set the SECURITY parameter to INVOKER, e.g...
+
+```
+CREATE OR REPLACE SQL SECURITY INVOKER 
+VIEW `view_name` AS 
+SELECT 
+```
+
+### MySQL server has gone away
+
+Errors such as "PDO Exception 'MySQL server has gone away'" are usually simply the result of exhausting your existing diskspace. Be sure you have sufficient space allocated to the service in [.platform/services.yaml](/configuration/services.md).
+
+The current disk usage can be checked using the CLI command `platform db:size`. It is usually advisable to have at least double the reported size available.
```

Comment:
We should warn that command is not all that reliable.  I don't know what you mean by "advisable to have at least double the reported size available".  You should make sure you have twice the space available it tells you is available?  That's an infinite loop. :smile: 

This is also a good place to link people to the health notifications, which is the better solution.